packages/next/build/webpack/plugins/wellknown-errors-plugin/parseNotFoundError.ts

```diff
@@ -107,6 +114,15 @@ export async function getNotFoundError(
       importTrace() +
       '\nhttps://nextjs.org/docs/messages/module-not-found'
 
+    if (await isEdgeRuntimeCompiled(compilation, input.module, config)) {
+      const moduleName = getUnresolvedModuleFromError(input.message)
+      if (moduleName) {
```

Comment:
```suggestion
    const moduleName = getUnresolvedModuleFromError(input.message)
    if (moduleName) {
      if (await isEdgeRuntimeCompiled(compilation, input.module, config)) {
```

It might be a little bit faster to swap these lines...

Reply:
thanks